src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/Replies.java

```diff
@@ -1,9 +1,16 @@
 package com.wikia.webdriver.elements.mercury.components.discussions.common;
 
+import com.google.common.base.Predicate;
+import com.wikia.webdriver.common.core.CommonExpectedConditions;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.common.core.CommonExpectedConditions'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)